extension/parquet/parquet_reader.cpp

```diff
@@ -155,7 +158,8 @@ static LogicalType DeriveLogicalType(const SchemaElement &s_ele) {
 
 static unique_ptr<ColumnReader> CreateReaderRecursive(ParquetReader &reader, const FileMetaData *file_meta_data,
```

Comment:
Perhaps we can make these functions members of the `ParquetReader` class instead of static, so the reader options do not need to be passed in?